Lesson8_Dictionary/homework8_dictionary.py

```diff
@@ -0,0 +1,50 @@
+# Task 1
+print('Task 1: \'Roles and Users\'')
+roles = {
+    'admin': ['Olga', 'Julia'],
+    'maintainer': ['Oleg', 'Roman'],
+    'manager': ['Ruslan', 'Artem'],
+    'developer': ['Viktor', 'Alex']
+}
+
+name = input('Please to enter name with upper registr for first letter: ')
+role = []
+for key, value in roles.items():
+    if name in value:
+        role = key
+        break
+if role:
+    print(f'Hello, {role}')
+else:
+    print('Hello, Guest')
+
+# Task 2
+print('\nTask 2: \'Formatted link\'')
+# result_link = 'https://www.youtube.com/watch?v=some_id&list=some_list&index=6&t=0s'
+params = {'v': 'some_id', 'list': 'some_list', 'index': '6', 't': '0s'}
+initial_str = 'https://www.youtube.com/watch?'
+result_str = ''
+
+for key, value in params.items():
+    result_str += f'{key}={value}&'
+result_str = result_str.rstrip('& ')
+result_link = initial_str + result_str
+print(f'After formatting the string and dictionary, we get this link: {result_link}')
+
```

Comment:
Can be better)
![image](https://github.com/annasever/python_cources/assets/51053483/e344b81a-afbd-4718-92a6-ad2945b01fb8)